resources/META-INF/plugin.xml

```diff
@@ -261,6 +261,9 @@
 
   <!-- Contributes IDEA-specific features and implementations. -->
   <depends optional="true" config-file="idea-contribs.xml">com.intellij.modules.java</depends>
+  <!-- Contributes Android Studio-specific features and implementations. -->
+  <!-- depends optional="true" config-file="studio-contribs.xml">com.intellij.modules.androidstudio</depends -->
```

Comment:
The commented-out line is the documented way to depend on android studio. It doesn't work for 3.0 beta. It may work in the future; I left the comment so we don't have to look up the name again.